pkg/backend/backend.go

```diff
@@ -109,6 +109,12 @@ type ListStacksFilter struct {
 	TagValue     *string
 }
 
+// ContinuationToken is an opaque string used for paginated backend requests. If non-nil, means
+// there are more results to be returned and the continuation token should be passed into a
+// subsequent call to the backend method. A nil continuation token means all results have been
+// returned.
+type ContinuationToken *string
```

Comment:
does this need to be a `*string`? I'd imagine that an empty string would be good enough to say "no continuation token".

Reply:
I went back and forth a lot on this. The empty string does seem reasonable enough to imply no continuation token. But I erred on keeping this a pointer because `if token == nil` seemed slightly preferable to `if token == ""`, IMHO.

But if you really don't like the design I can change it.